lib/notification/index.js

```diff
@@ -38,28 +38,28 @@ Notification.prototype = require("./apsProperties");
 Notification.prototype.headers = function headers() {
   let headers = {};
 
-  if (this.priority !== 10) {
-    headers["apns-priority"] = this.priority;
+  if (this.payload.priority) {
+    headers["apns-priority"] = this.payload.priority;
   }
 
-  if (this.id) {
-    headers["apns-id"] = this.id;
+  if (this.payload.id) {
+    headers["apns-id"] = this.payload.id;
   }
 
-  if (this.expiry > 0) {
-    headers["apns-expiration"] = this.expiry;
+  if (this.payload.expiry) {
+    headers["apns-expiration"] = this.payload.expiry;
   }
 
-  if (this.topic) {
-    headers["apns-topic"] = this.topic;
+  if (this.payload.topic) {
+    headers["apns-topic"] = this.payload.topic;
   }
 
-  if (this.collapseId) {
-    headers["apns-collapse-id"] = this.collapseId;
+  if (this.payload.collapseId) {
+    headers["apns-collapse-id"] = this.payload.collapseId;
   }
 
-  if (this.pushType) {
-    headers["apns-push-type"] = this.pushType;
+  if (this.payload.pushType) {
+    headers["apns-push-type"] = this.payload.pushType;
```

Comment:
I this could be done simpler by adding `pushType` to https://github.com/parse-community/node-apn/blob/master/lib/notification/index.js#L27

Reply:
I'm not that familiar with JS or this codebase, so I don't understand how this would play out; why it would work. I can take your word for it, but I dislike committing code I don't fully understand 😃

Can you explain this?